source/administration/object-management.rst

```diff
@@ -45,8 +45,8 @@ following:
          2020-01-02-MinIO-Advanced-Deployment-comments.json
          2020-01-04-MinIO-Interview.md
 
-MinIO supports multiple levels of nested directories and objects to support 
-even the most dynamic object storage workloads.
+MinIO supports multiple levels of nested directories and objects using a :term:`prefix` structure to support even the most dynamic object storage workloads.
+For the best performance, MinIO recommends that any single prefix contain no more than 10,000 objects.
```

Comment:
I think 10,000 is very small @djwfyi it can be easily 1million without issues on a large enough cluster. It is only a problem of smaller deployments that do not have a lot of disks.